lib/orderbook/OrderBook.ts

```diff
@@ -434,6 +442,12 @@ class OrderBook extends EventEmitter {
         this.pool.broadcastOrderInvalidation(removeResult.order, takerPubKey);
```

Comment:
Question: Are we sending invalidation even if !removeResult.fullyRemoved?

Reply:
I don't understand. why to send invalidation of the order is still valid (after partial swap)?

Reply:
It lets peers know that part of the order is no longer available, so they know to reduce the quantity on their end.

Reply:
not the best name for the operation.
As a minimum,  I think we should add a comment explaining what this method does.

In addition, while a P2P message was sent to multiple nodes, there is nothing in the logs (senders and receivers) which indicates that a message was sent accepted and processed (not to mention the outcome. Suggest you add this as part of #649 (or a new issue if you like).

Reply:
Good catch @offerm.

It's definitely confusing as of now.

Reply:
I'll add comments for the packet in a separate PR, but we do have that logging [when the packet is received and processed](https://github.com/ExchangeUnion/xud/blob/3868681741bada5ac39166152f14530ee3873f3a/lib/p2p/Pool.ts#L490) as well as logging when an [order is removed or reduced for any reason](https://github.com/ExchangeUnion/xud/blob/3868681741bada5ac39166152f14530ee3873f3a/lib/orderbook/TradingPair.ts#L206). Let's continue this in #649.

Reply:
Need to use verbose mode @offerm

Reply:
Verbose messages should be logged by default, since the default level is `debug` which logs everything except `trace`.